tests/cpp/test_multidevice_pipeline.cpp

```diff
@@ -126,7 +125,9 @@ void PipelineTest::executeAndValidate(bool validate_with_prescribed_values) {
   }
 
   runtime = std::make_unique<MultiDeviceExecutor>(
-      std::move(fusion), *communicator_, host_ir_executor_params);
+      std::make_unique<Fusion>(*fusion),
```

Comment:
Why this change? `fusion` doesn't seem to be used after.

Reply:
you're right, let me revert it.